Automattic/jetpack
# Boost: Update Contact Support button after Cloud CSS showstopper retry

Fixes 47-gh-Automattic/boost-shield-api

#### Changes proposed in this Pull Request:

* Update the "Contact Support" button link at the end of the Critical CSS Showstopper process to link to the Jetpack Support form for users with a premium plan.

#### Jetpack product discussion


NA

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Go to Jetpack Boost dashboard with Premium Plan
* Simulate a showstopper error. Eg:
  ```
  if ( ! empty( $_GET['jb-generate-critical-css' ] ) ) {
	header("HTTP/1.0 500 Internal Server Error");
	die();
  }
* Enable `Automatically Optimize CSS Loading`
* Confirm that the showstopper error is shown with `Refresh` button.
* Click the `Refresh` button to regenerate Critical CSS.
* Confirm that the showstopper error is shown again with `Contact Support` button.
* Confirm that clicking the `Contact Support` takes you to P1moTy-pc-p2
